selecting_reference_data.ipynb

```diff
@@ -53,6 +53,7 @@
     "    \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.            km = KMeans(n_clusters=1, random_state=42, n_init="auto") # Initializing kmeans model with 1 cluster
</pre><p>Since outliers are already removed, I don't think we need to use k-means to do this. Potentially something to add to future work. Also, I think k-means randomly initializes a centroid for the cluster, so the centroid might not be the mean</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>